### PR TITLE
Add 'timezone' field to channel

### PR DESCRIPTION
**Pull request by riteshshrv**

### CODE REVIEW

File: channel.py [hunk 1]
```diff
@@ -147,6 +148,11 @@ class SaleChannel(ModelSQL, ModelView):
         'Last Inventory Export Time', states=INVISIBLE_IF_MANUAL,
         depends=['source']
     )
+
+    timezone = fields.Selection(
+        TIMEZONES, 'Timezone',
+        translate=False, required=True
+    )
```
Comment (priyankarani) [inline channel.py]: default should be UTC

File: channel.py [hunk 2]
```diff
@@ -147,10 +148,19 @@ class SaleChannel(ModelSQL, ModelView):
         'Last Inventory Export Time', states=INVISIBLE_IF_MANUAL,
         depends=['source']
     )
+
+    timezone = fields.Selection(
```
Comment (priyankarani) [inline channel.py]: add this field to view too

**Comment by priyankarani** [comment]
LGTM

File: setup.py [hunk 3]
```diff
@@ -140,6 +140,7 @@ def run(self):
     %s = trytond.modules.%s
     """ % (MODULE, MODULE),
     test_suite='tests',
+    tests_require=['pytz'],
```
Comment (priyankarani) [inline setup.py]: Its not just required for tests

File: view/sale_channel_form.xml [hunk 4]
```diff
@@ -20,6 +20,8 @@
                 <page string="General" id="general">
                 </page>
                 <page string="Connection" id="connection">
+                    <label name="timezone"/>
+                    <field name="timezone"/>
```
Comment (priyankarani) [inline view/sale_channel_form.xml]: Move this to last import / export tab

File: setup.py (hunk 3 above)
Comment (riteshshrv) [inline setup.py]: test will fail without this

File: view/sale_channel_form.xml [hunk 5]
```diff
@@ -38,6 +38,8 @@
                     <field name="last_product_export_time"/>
                     <label name="last_inventory_export_time"/>
                     <field name="last_inventory_export_time"/>
+                    <label name="timezone"/>
+                    <field name="timezone"/>
```
Comment (priyankarani) [inline view/sale_channel_form.xml]: add colspan="3"

and move these to top of tab. means before <label name="last_order_import_time"/>

File: view/sale_channel_form.xml [hunk 6]
```diff
@@ -26,6 +26,8 @@
                 <page string="Advanced" id="advanced">
                 </page>
                 <page string="Last Import / Export Time" id="last_import_export_time" states="{'invisible': Eval('source') == 'manual' }">
+                    <label name="timezone"/>
```
Comment (tarunbhardwaj) [inline view/sale_channel_form.xml]: As this is external channel specific config this should be in connection tab only.

**Pull request merged**